.github/labeler-pr-triage.yml

```
@@ -668,6 +668,10 @@ service/route53resolver:
   - 'aws/internal/service/route53resolver/**/*'
   - '**/*_route53_resolver_*'
   - '**/route53_resolver_*'
+service/route53recoveryreadiness:
+  - 'aws/internal/service/route53recoveryreadiness/**/*'
+  - '**/*_route53_recovery_readiness_*'
+  - '**/route53_recovery_readiness_*'
```

We prefer the service part of the resource type name to be a single word (i.e. no - or _ characters) - this will aid with future automation.

```suggestion
  - '**/*_route53recoveryreadiness_*'
  - '**/route53recoveryreadiness_*'
```

---

Makes sense! Committed.